phc-rails/spec/models/event_registration_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe EventRegistration, type: :model do
+  describe "#to_salesforce_object" do
+    let(:account) { Account.create(sf_id: 123) }
+    let(:event_registration) do
+      EventRegistration.create({
+          account_id: account.id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.